packages/core/test/combinator/concatMap-test.js

```diff
@@ -96,18 +92,16 @@ describe('concatMap', function () {
 
   it('should dispose all inner streams', function () {
     const values = [1, 2, 3]
-    const spies = values.map(function () {
-      return spy()
-    })
+    const spies = values.map(() => spy())
 
-    const inners = values.map(function (x, i) {
+    const inners = values.map((x, i) => {
       return new Stream(new FakeDisposeSource(spies[i], streamOf(x).source))
     })
 
     const s = concatMap.concatMap(identity, fromArray(inners))
 
-    return drain(s).then(function () {
-      spies.forEach(function (spy) {
+    return drain(s).then(() => {
+      spies.forEach(spy => {
```

Comment:
hmm yeah thats tough, i can see it as a 1-liner.. but i guess its to the 👁  of the beholder haha in this case you can make the call 😄